lib/src/git.rs

```diff
@@ -2133,6 +2134,30 @@ fn expand_fetch_refspecs(
         .collect()
 }
 
+/// A list of refspecs that were ignored during a fetch. Callers should
+/// consider displaying these in the UI as appropriate.
+#[derive(Debug)]
+#[repr(transparent)]
+pub struct IgnoredRefspecs(pub Vec<IgnoredRefspec>);
+
+impl ops::Deref for IgnoredRefspecs {
+    type Target = Vec<IgnoredRefspec>;
```

Comment:
nit: We tend to avoid implementing `Deref` for newtypes because `Deref` is powerful. https://doc.rust-lang.org/stable/std/ops/trait.Deref.html